index.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset='utf8'>
```

Comment:
this part of the template always give error, the correct is `<meta chartset='utf-8'>`

Reply:
I always get tricked by it. 😄  Thank you for pointing it out @abelRoland . I will clean it up with the second batch of PR!